index.js

```diff
@@ -0,0 +1,59 @@
+let form = document.getElementById("new-task-form");
+let input = document.getElementById("new-task-input");
+let message = document.getElementById("message");
+let posts = document.getElementById("tasks");
+
+form.addEventListener("submit", (e) => {
+  e.preventDefault();
+  formValidation();
+});
+let formValidation = () => {
+  if (input.value === "") {
+    message.innerHTML = "Input field can not be blank...";
+  } else {
+    message.innerHTML = "";
+    acceptData();
+  }
+};
+let data = [];
+
+let acceptData = () => {
+  data.push({
+    text: input.value,
+  });
+  localStorage.setItem("data", JSON.stringify(data));
+  createPost();
+};
+let createPost = () => {
+  posts.innerHTML = "";
+  data.map((x, y) => {
+    return (posts.innerHTML += ` <div id=${y}><div id="tasks">
+    <div class="task">
+      <div class="content">
+        ${x.text}
+      </div>
+      <div class="actions">
+        <button  onClick="editPost(this)" class="edit">Edit</button>
+        <button onClick="deletePost(this)" class="delete">Delete</button>
+      </div>
+    </div> </div>`);
+  });
+
+  input.value = "";
+};
+
+let deletePost = (e) => {
+  e.parentElement.parentElement.remove();
+  data.splice(e.parentElement.parentElement.id, 1);
+  localStorage.setItem("data", JSON.stringify(data));
+};
+let editPost = (e) => {
```

Comment:
![image](https://github.com/milos135/TodoListJs/assets/85948457/ccc4f776-9d31-497d-a240-1b6858f2eec5)
there is space added when i want to edit task